public/locales/en-US/list.json

```diff
@@ -78,5 +78,7 @@
   "revokeTxn-sent-success": "Transaction sent success",
   "user-rejected": "User rejected request",
   "please-connect-mm": "Please connect to MetaMask first",
-  "switch_network_success": "Switch network success"
+  "switch_network_success": "Switch network success",
+  "waiting_for_tx": "Waiting for transaction",
+  "waiting_for_block": "Waiting for block"
```

Comment:
What's the difference between `waiting for tx` and `waiting for block`. I ask this because `tx` is usually confirmed along with a `block`.

Does `waiting for tx` mean `waiting for tx submission`?

Reply:
Yes, it's waiting for user to click on confirm, how abt `Waiting for confirm`?

Reply:
Got it and `Waiting for confirm` is fine except that `confirm` is a verb

Reply:
How abt change to confirmation